### ISSUE TITLE
[RFC] quick ssh key copy for salt-ssh [Document enhanced request]

### ISSUE THREAD

**Issue by netroby** [saltstack/salt#27393]
salt-ssh default not respect system /etc/ssh/ssh_config , so we need to copy key (generated by salt-ssh) to target machines.
the default key location at cat /etc/salt/pki/master/ssh/salt-ssh.rsa
If our target machines already do not allow login using user/password. we will unable to deploy public key using salt-ssh. we need add key by hand.

i wrote a small shell script to make things more easier

```
#!/bin/bash
if [ -z $1 ]; then
   echo $0 user@host.com
   exit 0
fi
ssh-copy-id -i /etc/salt/pki/master/ssh/salt-ssh.rsa  $1
```

If we could add this to salt-ssh document , it will save users time.

**Comment by basepi** [comment]
You can also specify a private key for salt to use if you already have access to a server with a given key. Just specify `priv: <path_to_key>` in your roster. Might be easier than manually deploying the salt-specific key.

Anyway, if you wanted to submit a pull request to the salt-ssh documentation with this script, including steps for how to use it, I think that would be a good addition.

**Comment by cachedout** [comment]
Closed via #27434
